src/raftstore/store/util.rs

```diff
@@ -509,6 +509,8 @@ pub enum LeaseState {
     Valid,
     /// The lease is expired.
     Expired,
+    /// Postponed lease check.
+    Postponed,
```

Comment:
I don't think it's a good idea. `Postponed` is not a state of lease itself, it's a state of the invoking procedure which decides to postponed lease check. Just like we can't introduce a `Postponed` role to raft node because we want to check if it's leader later.

Reply:
What do you think? @Hoverbear

Reply:
Okay, I will remove `Postponed`,